src/pykeen/pipeline.py

```diff
@@ -531,6 +531,8 @@ def pipeline(  # noqa: C901
     training_triples_factory: Optional[TriplesFactory] = None,
     testing_triples_factory: Optional[TriplesFactory] = None,
     validation_triples_factory: Optional[TriplesFactory] = None,
+    restrict_evaluation_to_entities: Optional[Collection[str]] = None,
```

Comment:
maybe `evaluation_entity_whitelist` might be more descriptive. Same for the relations on the next line

Reply:
fc26d54